src/code/views/node-view.tsx

```diff
@@ -410,10 +410,12 @@ getDefaultProps() {
           />
         );
       } else {
+        const outLinks = node.outLinks();
+        const hasAddedToLink = _.filter(outLinks, link => link.relation.formula === "+in").length > 0;
+        const hasSubtractedFromLink = _.filter(outLinks, link => link.relation.formula === "-in").length > 0;
```

Comment:
Could also use `_.some()` which short-circuits the iteration and eliminates the need for the length check.

Reply:
@kswenson I switch the code out to use `_.some(...)` after rebasing to the final phase-1 version.